InstallDebian.md

```diff
@@ -0,0 +1,83 @@
+# TDS Foreign data wrapper
+
+* **Author:** Geoff Montee
+* **Name:** tds_fdw
+* **File:** tds_fdw/InstallDebian.md
+
+## Installing on Debian
+
+This document will show how to install tds_fdw on Debian 10. Other Debian distributions should be similar.
+
+### Install FreeTDS
+
+The TDS foreign data wrapper requires a library that implements the DB-Library interface,
+such as [FreeTDS](http://www.freetds.org).
+
+```bash
+sudo apt-get install libsybdb5 freetds-dev freetds-common
+```
+
+### Install PostgreSQL
+
+If you need to install PostgreSQL, do so by following the [apt installation directions](https://wiki.postgresql.org/wiki/Apt). For example, to install PostgreSQL 11 on Debian:
+
+```bash
+sudo bash -c "echo \"deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main\" > /etc/apt/sources.list.d/pgdg.list"
```

Comment:
There's a more efficient way of doing this on Debian, since Debian 9:
```
sudo bash -c 'source /etc/os-release; echo "deb http://apt.postgresql.org/pub/repos/apt/ ${VERSION_CODENAME}-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
```
I strongly suggest we change to this, as `lsb_release` (part of `lsb-release` package) is not available at Docker images, and installing it installs a lot of dependencies.